spec/support/seeds/seeds_e2e.rb

```diff
@@ -71,6 +71,33 @@
 
 ##############################################################################
 
+seeder.create_if_doesnt_exist(User, "email", "moderator-user@forem.local") do
```

Comment:
I think you are running into some similar issues as @msarit did with the E2E test failures - there's a spec that assumes the 'trusted-user-1@forem.local' below is user with `id: 2`, but after adding this one they're `id: 3` 🙃 

I mentioned to Arit that probably that test wants reworked so it doesn't rely on the ID, but I think for a quick fix if you create this user _after_ the one immediately below you should see some of those specs just magically start passing again

Reply:
Thank you Suzanne! 🙏🏾